spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/PubSubTemplate.java

```diff
@@ -124,4 +160,11 @@ public void setMessageConverter(MessageConverter messageConverter) {
 		this.messageConverter = messageConverter;
 	}
 
+	public int getExecutorThreadCount() {
+		return this.executorThreadCount;
+	}
+
+	public void setExecutorThreadCount(int executorThreadCount) {
+		this.executorThreadCount = executorThreadCount;
```

Comment:
I'd like to support both: specifying both `executorThreadCount` and `executorProvider`.

I'm not sure how acceptable it would be to allow Spring users to specify Google Cloud Pub/Sub stuff into the template. Perhaps we could make use of `PubSubAdmin` in #26 to help us create PubSub executors? Then we'd have Spring idioms only.